src/snowflake/telemetry/_internal/exporter/otlp/proto/log_exporter/__init__.py

```diff
@@ -0,0 +1,170 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import abc
+import enum
+import logging
+import logging.config
+import threading
+import typing
+import opentelemetry.sdk.util.instrumentation as otel_instrumentation
+
+from opentelemetry.sdk import resources
+from opentelemetry.sdk._logs import export
+from opentelemetry.sdk import _logs
+from opentelemetry.util import types
+from snowflake.telemetry._internal.encoder.otlp.proto.common.log_encoder import (
+    serialize_logs_data
+)
+
+LOGGER_NAME_TEMP_ATTRIBUTE = "__snow.logging.temp.logger_name"
+FILEPATH_ATTRIBUTE = "code.filepath"
+FUNCTION_NAME_ATTRIBUTE = "code.function"
```

Comment:
Should we prefix with "_" to signal that they are internal?

Reply:
Done, also moved inside the classes they need them as members instead of globals.